soc/arm/nxp_lpc/lpc55xxx/Kconfig.soc

```diff
@@ -116,6 +116,10 @@ config SOC_PART_NUMBER_LPC55XXX
 config INIT_PLL0
 	bool "Initialize PLL0"
 
+config INIT_PLL1
+	bool "Initialize PLL1"
+	default "y"
```

Comment:
Since it seems that enabling this Kconfig will clock the SOC at 150MHz, I'd like to see a help string clarifying this effect.

Also, should this KConfig depend on `!SOC_LPC55S06`, since it seems that part does not use PLL1?